### ISSUE TITLE
[Bug] [a11y]Onboarding screen - Touch target items need to be made larger

### ISSUE THREAD

**Issue by Diana-Rus** [mozilla-mobile/fenix#9583]
## Prerequisites:
- Have Google Accessibility Scanner installed.
- Have a clean Firefox profile.

### Steps to reproduce
1. Open app and inspect the screen elements with the scanner.
### Expected behavior
There are no suggestions to make any changes.
### Actual behavior
Section: See what's new -> "Get answers here" link
- org.mozilla.fenix.nightly:id/get_answers
This item's height is 43dp. Consider making the height of this touch target 48dp or larger.

Section: Protect yourself -> Enable/Disable ETP toggle
- org.mozilla.fenix.nightly:id/tracking_protection_toggle
This item's size is 47dp x 27dp. Consider making this touch target 48dp wide and 48dp high or larger.

Section: Protect yourself
 -> "Standard" radio button & "Strict" radio button
- org.mozilla.fenix.nightly:id/tracking_protection_standard_option
- org.mozilla.fenix.nightly:id/tracking_protection_strict_default
This item's size is 40dp x 32dp. Consider making this touch target 48dp wide and 48dp high or larger.

Section: Choose your theme
-> "Light theme" radio-button & "Dark theme" radio button
- org.mozilla.fenix.nightly:id/theme_light_radio_button
- org.mozilla.fenix.nightly:id/theme_dark_radio_button
This item's size is 32dp x 32dp. Consider making this touch target 48dp wide and 48dp high or larger.
-> "Automatic" radio-button
-org.mozilla.fenix.nightly:id/theme_automatic_radio_button
This item's size is 40dp x 32dp. Consider making this touch target 48dp wide and 48dp high or larger.

Section Browse privately -> "Open Settings" button
- org.mozilla.fenix.nightly:id/open_settings_button
This item's height is 18dp. Consider making the height of this touch target 48dp or larger.

Section: Take a position
-> Toolbar "Top"  & "Bottom" options
- org.mozilla.fenix.nightly:id/toolbar_top_radio_button
- org.mozilla.fenix.nightly:id/toolbar_bottom_radio_button
This item's size is 32dp x 32dp. Consider making this touch target 48dp wide and 48dp high or larger.

Section: Your pivacy
-> Read our privacy notice
- org.mozilla.fenix.nightly:id/read_button
This item's height is 38dp. Consider making the height of this touch target 48dp or larger.

### Device information

* Android device: Samsung Galaxy S9 (Android 8)
* Fenix version: Nightly 4/1 (Build #20920607)

**Screenshots:**
**Section: See what's new -> "Get answers here" link**
![Screenshot_20200401-122702_Accessibility Scanner](https://user-images.githubusercontent.com/49859863/78123614-198d9380-7417-11ea-8bb4-97d6ae57f239.jpg)

**Section: Protect yourself -> Enable/Disable ETP toggle**
![Screenshot_20200401-122718_Accessibility Scanner](https://user-images.githubusercontent.com/49859863/78123556-02e73c80-7417-11ea-8f03-656dcbfb6068.jpg)

**Section: Protect yourself -> "Standard" radio button**
![20200401_123053](https://user-images.githubusercontent.com/49859863/78123594-0f6b9500-7417-11ea-80f8-79a08adc84e4.jpg)

Section: Choose your theme
-> "Light theme" radio-button & "Dark theme" radio button
![Screenshot_20200401-142348_Accessibility Scanner](https://user-images.githubusercontent.com/49859863/78132673-d4249280-7425-11ea-94e9-0e5e7afaa683.jpg)

-> "Automatic" radio-button
![Screenshot_20200401-142550_Accessibility Scanner](https://user-images.githubusercontent.com/49859863/78132652-cbcc5780-7425-11ea-9421-ddd969f47ac4.jpg)

Section Browse privately -> "Open Settings" button
![Screenshot_20200401-142754_Accessibility Scanner](https://user-images.githubusercontent.com/49859863/78132629-c0792c00-7425-11ea-861c-dd1f66be2db7.jpg)

Section: Take a position
-> Toolbar "Top" options
![Screenshot_20200401-143019_Accessibility Scanner](https://user-images.githubusercontent.com/49859863/78132906-2f568500-7426-11ea-8b8d-281aec2a014b.jpg)

Section: Your pivacy
-> Read our privacy notice
![Screenshot_20200401-143039_Accessibility Scanner](https://user-images.githubusercontent.com/49859863/78132913-341b3900-7426-11ea-9770-e71d81ae64f1.jpg)

**Comment by asadotzler** [comment]
I don't think we need to worry about the size of the touch target for radio buttons that overlay the top left corner of a picture because the whole picture is a target for the same action. Other targets should be adjusted.

**Comment by sblatz** [comment]
@mcarare having them be *clickable* at 48dp is totally fine, but let's not update the visual height :)

**Comment by sflorean** [comment]
Tested on Nightly 6/4 with Google Pixel 3(Android 10) and LG G7 FIT (Android 8) and I can confirm that this issue is fixed. However, the buttons for Protect yourself card: Standard and Strict have 40dp x 32dp, same as before. @mcarare could you please confirm that this is expected? Thank you!

**Comment by mcarare** [comment]
@sflorean This is expected, and I did not intended to change that. They are in the same situation as elements mentioned in https://github.com/mozilla-mobile/fenix/issues/9583#issuecomment-609987250, because their touch target extends to the policy title and description.
